.github/workflows/check-labels.yml

```diff
@@ -11,11 +11,10 @@ on:
     branches: [main]
     paths-ignore: [.github]
 
-  # To allow testing PRs that change workflows.
+  # To check labels on ghstack PRs and to allow testing PRs that change workflows.
   # May be triggered together with pull_request_target, it's OK.
   pull_request:
     types: [opened, synchronize, reopened, labeled, unlabeled]
-    paths: [.github]
```

Comment:
Do you mind adding `branches-ignore: [main]` here, otherwise it's possible the same workflow will be scheduled twice?

Reply:
Then there will be no way to test changes to the workflow itself with a normal PR.

Reply:
Added with a comment.